.github/workflows/stale.yml

```diff
@@ -0,0 +1,37 @@
+name: Reusable Stale
+
+on:
+  workflow_call:
+    inputs:
+      days-before-close:
+        required: false
+        default: 42
```

Comment:
42 days seems to be large. Can we not close earlier?

Reply:
We can certainly do that. I myself think it is too long but I based it off [this](https://github.com/studyportals/SearchPage/blob/2158b19b3fc7d0189d4471c05c340c9d6c430b5c/.github/stale.yml)

Reply:
Change this
```suggestion
        default: 46
```

Reply:
from https://knowledge-vault.studyportals.xyz/services-and-tools/tools/dependabot.html#stalebot

Reply:
Done